src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Package/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Package.csproj

```diff
@@ -4,7 +4,6 @@
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <IsPackable>true</IsPackable>
     <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
This isn't needed anymore?

Reply:
No - we doa separate pack command. I think this is in the template so people don't need to do both a build and pack. It isn't necessary and was causing some build errors on the ci